File: API/RecipeAPI/Models/Recipes.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace RecipeAPI.Models
{
    public class Recipes
    {
        [Key]
        public int Recipe_id { get; set; }

        [Required]
        public String RecipeName { get; set; }

        [Required]
        public String Difficulty { get; set; }

        [Required]
        public double Duration { get; set; }
        //Change so its like ca with Rtae and Blood App

        public override string ToString()
        {
            return RecipeName + " " + Difficulty + " " + Duration;
        }
    }
}

File: app/src/main/java/com/example/androidfrontend/Recipes.java
package com.example.androidfrontend;

public class Recipes {

    private String openLibraryId;

    private int Recipe_id;
    private String RecipeName;
    private String Difficulty;
    private double Duration;


    public String getOpenLibraryId() {
        return openLibraryId;
    }

    public String getRecipeName() {
        return RecipeName;
    }

    public int getRecipe_id(){

        return Recipe_id;
    }
    public String getDiff(){

        return Difficulty;
    }
    public double getDuration(){

        return Duration;
    }

    public String toString()
    {
        return RecipeName + " " + Difficulty + Duration;
    }


}
